### PR TITLE
Grape formatter feature requested in #1268

### PR DESCRIPTION
**Pull request by jpaas**
for issue #1258 

re:  #1268

### CODE REVIEW

File: lib/grape-active_model_serializers.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# To add graoe support, require 'grape-active_model_serializers' in the base of your grape endpoints
```
Comment (bf4) [inline lib/grape-active_model_serializers.rb]: typo s/graoe/grape

should be `active_model_serializers/grape` maybe?

Comment (bf4) [inline lib/grape-active_model_serializers.rb]: or you think it should be in the grape namespace?

**Comment by bf4** [comment]
:100: awesome work!

File: lib/grape-active_model_serializers.rb (hunk 1 above)
Comment (jpaas) [inline lib/grape-active_model_serializers.rb]: I can do that. I'll update the request.

**Comment by bf4** [comment]
@jpaas make sure you run rubocop to fix the ci failures. `rake rubocop:auto_correct` is often sufficient, but always review any changes it makes :)

 e.g.


```plain
Running RuboCop...
Inspecting 100 files
.....................................C.C............................................................
Offenses:
lib/grape/active_model_serializers.rb:6:49: C: Style/TrailingBlankLines: Final newline missing. (https://github.com/bbatsov/ruby-style-guide#newline-eof)
require 'grape/helpers/active_model_serializers'
                                                
lib/grape/helpers/active_model_serializers.rb:16:4: C: Style/TrailingBlankLines: Final newline missing. (https://github.com/bbatsov/ruby-style-guide#newline-eof)
end
```

**Comment by jpaas** [comment]
added rubocop corrections to pull request

**Comment by bf4** [comment]
@jpaas would you mind rebasing and repushing? I'm not sure why it's marked as failed.

**Comment by bf4** [comment]
bump see https://github.com/rails-api/active_model_serializers/pull/1273#issuecomment-155375042

**Comment by bf4** [comment]
@jpaas Thanks for #1336 . Next time, please just for push to the same remote so we don't have duplicate prs with comments that stretch across them. :)

**Comment by johnhamelink** [comment]
FYI @bf4 it was me who pushed to the new PR - I can't push to this PR because it belongs to @jpaas

**Comment by bf4** [comment]
@johnhamelink woops :)

**Pull request closed without merging**